xml/tuning_dynamic_debug.xml

```diff
@@ -0,0 +1,209 @@
+        <listitem>
+          <para>
+            To enable the debug messages for a specific file, use the following
+            example:
+          </para>
+<screen>&prompt.root;<command>echo "file <replaceable>FILE_NAME.c</replaceable> +p" > /sys/kernel/debug/dynamic_debug/control</command></screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Enabling debug messages for a specific module</term>
+        <listitem>
+          <para>
+            To enable debug messages for a specific kernel module, use the
+            following example:
+          </para>
+<screen>&prompt.root;<command>echo "module <replaceable>MODULE_NAME</replaceable> +p" > /sys/kernel/debug/dynamic_debug/control</command></screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Enabling debug messages with a specific log level</term>
+        <listitem>
+          <para>
+            To enable debug messages with a specific log level, use the
+            following example:
+          </para>
+<screen>&prompt.root;<command>echo "format \"<replaceable>LOG_LEVEL</replaceable>\" +p" > /sys/kernel/debug/dynamic_debug/control</command></screen>
+          <warning>
+            <title>Avoid excessive logging of debug messages</title>
+            <para>
+              Dynamic debug should be used judiciously, as excessive logging
+              can impact system performance and fill up log files. It is
+              important to disable debugging messages when they are no longer
+              needed.
+            </para>
+          </warning>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Disabling debug messages</term>
+        <listitem>
+          <para>
+            To disable previously enabled debugging messages for a specified
+            file or kernel module, run the <command>echo</command> command with
+            the <command>-p</command> option. For example:
+          </para>
+<screen>&prompt.root;<command>echo "<replaceable>FILE_NAME.c</replaceable> -p" > /sys/kernel/debug/dynamic_debug/control</command></screen>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <para>
+      For detailed information about dynamic debug and its use cases, refer to
+      its
+      <link xlink:href="https://www.kernel.org/doc/html/latest/admin-guide/dynamic-debug-howto.html">official
+      documentation</link>.
+    </para>
+  </sect1>
+  <sect1 xml:id="sec-tuning-dynamic-debug-view">
+    <title>Viewing the dynamic debug messages</title>
+
+    <para>
+      You can view the dynamic debug messages that were generated based on the
+      configurations you enabled, by running <command>dmesg</command> and
+      filtering the output with <command>grep</command>. For example:
+    </para>
+
+<screen>&prompt.root;<command>dmesg | grep -i "<replaceable>FILE_NAME.c</replaceable>"</command></screen>
```

Comment:
At least in the PDF file this like is broken into 2 lines; I see this:
```
# dmesg | grep -i
"FILE_NAME.c"
```
Can that be fixed?